src/board/devices/input/rotarySwitch.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Represents the rotary switch in memory.
+ *
+ * @author Philippe Schneider
+ */
+
+import { Device } from 'board/devices/device'
+import { Byte, Word } from 'types/binary'
+
+export class RotarySwitch extends Device {
+  private RotarySwitch_ADDRESS = Word.fromUnsignedInteger(0x60000211)
+  private maxValue = 255
+  private minValue = 0
+
+  public isReadOnly = false
+  public isVolatile = false
+  public startAddress = this.RotarySwitch_ADDRESS
+  public endAddress = this.RotarySwitch_ADDRESS
+
+  /**
+   * increase the rotary switch
+   *
+   */
+  public increase(): void {
+    let newValue = this.memory.readByte(this.RotarySwitch_ADDRESS).value + 1
```

Comment:
Instead of reading the `value` from the byte type you could also use the `add()` function directly on the `Byte` object to increase the value and then there is no need to call `fromUnsignedInteger()` below again to convert the value to a `Byte`.

Reply:
Thank you, changed it.